eng/Versions.props

```diff
@@ -7,8 +7,8 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <SdkBandVersion>6.0.100</SdkBandVersion>
-    <PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>2</PreReleaseVersionIteration>
+    <PreReleaseVersionLabel>rtm</PreReleaseVersionLabel>
+    <PreReleaseVersionIteration></PreReleaseVersionIteration>
```

Comment:
@danmoseley, these rebates aren't doing the right things...

Reply:
I suspect it would be easier/cleaner to back out (with a force push) the rebase commit, and change this back to target `release/6.0`, merge this as it was into that branch, and then use the backport bot to port from `release/6.0` to `release/6.0-rc2`.

Reply:
I think we could cherry-pick Adam and Tom's commits and force-push to release/6.0-rc2; then the bot that syncs rc2 with release/6.0 will port automatically.

Reply:
Oh wow. I should have noticed what this did. I wonder how many others...

Yes into 6.0 first is fine it just needs to be in RC2 tomorrow. I'm hesitant to attempt to fix anything again..

Reply:
To be clear, I rebased to include only the relevant commits and is already targeting RC2. No need to manually port to 6.0 as that is made automatically AFAIK.